nibabel/streamlines/tests/test_tractogram.py

```diff
@@ -18,6 +18,7 @@
 
 def setup():
     global DATA
+    DATA['rng'] = np.random.RandomState(1234)
```

Comment:
Unless nosetest sets the seed of np.random I prefer having reproducible tests. In practice, it should work with any seed, so there is no reason really. We also use a seed in `test_array_sequence.py`.